tests/modules/biobambam/bammerge/main.nf

```diff
@@ -0,0 +1,18 @@
+#!/usr/bin/env nextflow
+
+nextflow.enable.dsl = 2
+
+include { BIOBAMBAM_BAMMERGE } from '../../../../modules/biobambam/bammerge/main.nf'
+
+workflow test_biobambam_bammerge {
+
```

Comment:
Would be nice to also implement the test fort single-end files

Reply:
Apologies, it does work. I was a bit confused as to what was happening. My bad!

Reply:
done!